ee/codegen/src/__test__/__snapshots__/generate-code.test.ts.snap

```diff
@@ -104,6 +104,47 @@ class WorkflowDisplay(BaseWorkflowDisplay[Workflow]):
 "
 `;
 
+exports[`generateCode > should generate code for %1 integration-trigger-with-entrypoint-and-outgoing-edges.ts > triggers/slack_message_trigger.py 1`] = `
+"from vellum.workflows.triggers import IntegrationTrigger
+
+
+class SlackMessageTrigger(IntegrationTrigger):
+    message: str
+
+    class Config(IntegrationTrigger.Config):
+        provider = "COMPOSIO"
+        integration_name = "slack"
+        slug = "SLACK_MESSAGE_TRIGGER"
+
+    class Display(IntegrationTrigger.Display):
+        label = "Slack Message Trigger"
+        x = 0
+        y = 0
+        z_index = 0
+        icon = "slack"
+        color = "#4A154B"
+"
+`;
+
+exports[`generateCode > should generate code for %1 integration-trigger-with-entrypoint-and-outgoing-edges.ts > workflow.py 1`] = `
+"from vellum.workflows import BaseWorkflow
+
+from .nodes.custom_node import CustomNode
+from .nodes.final_output import FinalOutput
+from .triggers.slack_message_trigger import SlackMessageTrigger
+
+
+class Workflow(BaseWorkflow):
+    graph = {
+        CustomNode >> FinalOutput,
+        SlackMessageTrigger >> CustomNode,
```

Comment:
[oos] we likely want this to evolve into `{ManualTrigger, SlackMessageTrigger} >> CustomNode >> FinalOutput` in the future